packages/features/ee/workflows/components/AddVariablesDropdown.tsx

```diff
@@ -38,7 +38,7 @@ export const AddVariablesDropdown = (props: IAddVariablesDropdown) => {
               key={variable}
               type="button"
               className="px-5 py-1"
-              onClick={() => props.addVariable(props.isEmailSubject, variable)}>
+              onClick={() => props.addVariable(props.isEmailSubject, t(`${variable}_workflow`))}>
```

Comment:
Fixes bug that variables were always added in English and not the user's language